libraries/AP_GPS/AP_GPS.h

```diff
@@ -157,9 +174,8 @@ class AP_GPS
         return status(primary_instance);
     }
 
-    // Query the highest status this GPS supports
+    // Query the highest status this GPS supports (always reports GPS_OK_FIX_3D for the blended GPS)
```

Comment:
The only reason I think that maybe the RTK should be audited is for any future user. We wouldn't ever send the blended instance anyways because it would never be index 0 or 1, which is the required index for those two.